modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +45,41 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def find_or_create_ews(bgs_claim)
+          ews = ClaimsApi::EvidenceWaiverSubmission.find_by(claim_id: bgs_claim[:bnft_claim_dto][:bnft_claim_id])
+
+          unless ews&.status&.in?(%w[submitted pending])
+            attributes = {
+              status: ClaimsApi::EvidenceWaiverSubmission::PENDING,
+              auth_headers_ciphertext: auth_headers,
+              cid: source_cid
+            }
+
+            ClaimsApi::EvidenceWaiverSubmission.create!(attributes)
+          end
+        end
+
+        def attempt_background_submit(bgs_claim, ews)
```

Comment:
Concern: We'll need to handle the error messages and failure count updates here for the new columns. Do we want a separate ticket for that?

Reply:
i vote new ticket, this PR is already "too large/too many lines"